### DISCUSSION TITLE
Hope to increase the support for props

### DISCUSSION THREAD

**Discussion by skynice** [FranckFreiburger/vue3-sfc-loader#84] [Ideas]
Hope to increase the support for props

**Comment by FranckFreiburger** [comment]
please give more details

**Comment by skynice** [comment]
for examples,the compoment use [name] props,```loadModule()``` injection the [name] value.
```javascript
/* <!-- */
const sfcContent = `
<template>
  Hello {{name}} !
</template>
<script>
export default {
  props:{
    name:String
  }
}
</script>
`;
/* --> */
//...
loadModule('test.vue',{name:'World'},options);
```

**Reply by FranckFreiburger** [reply]
I do not plan to pass props through `loadModule()`.

However, you can pass data through the `moduleCache`, see the following example: https://github.com/FranckFreiburger/vue3-sfc-loader/blob/main/docs/examples.md#vue2-basic-example

If it does not meet your needs, please give more details about your usecase.
